### ISSUE TITLE
".../node_modules/tns-core-modules/ui/core/bindable/bindable"' has no exported member 'Bindable'

### ISSUE THREAD

**Issue by kumaran-dena** [rhanb/nativescript-bottombar#39]
If i tried with version 2.1.4 of nativescript-bottombar plugin with NS Cli version 3.2.1, then i have encountered with ".../node_modules/tns-core-modules/ui/core/bindable/bindable"' has no exported member 'Bindable' error.

FYI,
`PS D:\IFS\IFS_Latest\IFSAppMobile> tns plugin add nativescript-bottombar@2.1.4 D:\IFS\IFS_Latest\IFSAppMobile -- nativescript-bottombar@2.1.4 Successfully installed plugin nativescript-bottombar.`

After app builds successfully. But when i tried to run app, "..../bindable"' has no exported member 'Bindable' error occurred.

`PS D:\IFS\IFS_Latest\IFSAppMobile> tns run android
Searching for devices...
Executing before-liveSync hook from D:\IFS\IFS_Latest\IFSAppMobile\hooks\before-liveSync\nativescript-a
ngular-sync.js
Executing before-prepare hook from D:\IFS\IFS_Latest\IFSAppMobile\hooks\before-prepare\nativescript-dev
-typescript.js
Found peer TypeScript 2.2.2
node_modules/nativescript-bottombar/index.d.ts(3,10): error TS2305: Module '"D:/IFS/IFS_Latest/IFSAppMobile
/node_modules/tns-core-modules/ui/core/bindable/bindable"' has no exported member 'Bindable'.

Unable to apply changes on device: 3004c567d59f1200. Error is: TypeScript compiler failed with exit code 1.
Executing after-watch hook from D:\IFS\IFS_Latest\IFSAppMobile\hooks\after-watch\nativescript-dev-types
cript.js
PS D:\IFS\IFS_Latest\IFSAppMobile>`

Why this issue occurred!?

**Comment by rhanb** [comment]
Read the installation section again, your are using NS3 not NS2.

**Comment by kumaran-dena** [comment]
@rhanb Yes. i got it. 
If i tried nativescript-bottombar@3.0.4 with Nativescript Cli version 3.2.1, then i have encountered with Unfortunately app has stopped issue!

Regarding this issue,

- I have raised the topic [here.](https://discourse.nativescript.org/t/after-update-nativescript-cli-3-2-1-unfortunately-app-has-stopped-occurred/2764)

- I have raised the new [issue](https://github.com/NativeScript/nativescript-angular/issues/1028) on Nativescript-angular github repository.

Finally i have came out from issue by installed the nativescript-bottombar@3.0.5 (instead of 3.0.6).
